### ISSUE TITLE
Prepare for resourcing visual and UX design help in upcoming sprints

### ISSUE THREAD

**Issue by jameshupp** [18F/cg-product#626]
In order to produce well-researched and -explored input to front end design and engineering improvements while our dedicated design staff is light, we should align our needs and the design team's resourcing process in advance of needing this work.

Criteria:
- [x] cloud.gov product lead understands the process for getting designers to participate in short-term cloud.gov work.
- [ ] Backlog of upcoming cloud.gov work that might need designers: stories, ballpark workload, etc.
- [ ] Document the process within appropriate cloud.gov squad documents.

**Comment by nikzei** [comment]
[@James Hupp - TEACD] (1) I feel like this was in progress last sprint and (2) do we have more to do with this?

**Comment by brittag** [comment]
Just as an update - we ended up chatting a bit with the visual design team via Jen T, and I ended up talking to Amber briefly today about this. A couple of visual designers are likely to be available for small projects.

James and I chatted about the [Rates page](https://cloud.gov/overview/pricing/rates/) as a candidate for visual design help - it's super important and needs to communicate complex information.
